casual/casual-jca/src/main/java/se/laz/casual/jca/jmx/CasualMBean.java

```diff
@@ -0,0 +1,9 @@
+package se.laz.casual.jca.jmx;
+
+import java.util.List;
+
+public interface CasualMBean
```

Comment:
It useful to expose runtime information, in this case regarding the new network pools.
I am using it myself while running stuff on wildfly.
Not sure what you mean by determining some form of hierarchy?

Reply:
The data is just static data though - can't change after deployment scripts are run can it?

it's been a while since i played with JMX Beans, but i was thinking how it looks within the tree structure of jconsole - everything grouped in the tree appropriately based on function etc. So i thought do we want to take a beat and think what we want to have in general in the future and try and get these ones in the right place - or can we wait with these until later.

Reply:
That data is very much alive when physical connections come and go.

Reply:
Re the grouping, that's namespace based.

Reply:
I can add a screen shot if you want.